lib/undercover/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Undercover
-  VERSION = '0.2.2'
+  VERSION = '0.3.2'
```

Comment:
That could become just `0.3.0`, but I would suggest to revert this change for now. Could you add an entry to "unreleased" in `CHANGELOG.md` and I'll do the version bump with a release commit?